plugins/json_importer.cc

```diff
@@ -227,7 +298,7 @@ struct JSONImporter: public Procedure {
         {
             auto & threadAccum = accum.get();
 
-            int64_t actualLineNum = lineNumber + lineOffset;
```

Comment:
int64_t is better here.  It is possible that a 32 bit machine may have more than 4 billion lines.